### DISCUSSION TITLE
Help contribution

### DISCUSSION THREAD

**Discussion by runtimevic** [frangoteam/FUXA#1049] [General]
Hello,
I found this project, first of all, congratulations for this work.
I have created an organization on GitHub:
https://github.com/VisualPLC
I would like to create VPL visual programming language Blockly, etc. and low code tools for the world of industrial automation and also so that children can enter the world of programming,
I see that you have communication with different devices, mqtt, modbus, siemens step7, etc. and the project is created with node.js, if you are interested in the project and want to collaborate it would be of great help, so you can share and learn along the way and also help the community...
I am not an expert in node.js and I would really like to receive help creating communication with different devices...
many thanks for everything, Víctor.

**Comment by unocelli** [comment]
Hi, very interesting project, I am a big fan of Blockly. currently as you can see we have few resources. have you considered integrating Blockly into FUXA as an option for scripts?

**Comment by runtimevic** [comment]
> Hi, very interesting project, I am a big fan of Blockly. currently as you can see we have few resources. have you considered integrating Blockly into FUXA as an option for scripts?
Hello @unocelli,
I'm trying node.js for backend and react.js as frontend, but I'm not that expert and I'm looking for help and someone who wants to collaborate...
FUXA is developed with NodeJS (backend) and Angular (frontend).
I don't know Angular, and I'm starting with react or I'm open to other possibilities but I'm a newbie as a frontend...
